lib/ansible/modules/windows/win_hosts.py

```diff
@@ -18,62 +18,71 @@
 short_description: Manages hosts file entries on Windows.
 description:
   - Manages hosts file entries on Windows.
-  - Maps IPv4 or IPv6 addresses to canonical names
-  - Adds, removes, or sets cname records for ip and hostname pairs
-  - Modifies %windir%\system32\drivers\etc\hosts.
+  - Maps IPv4 or IPv6 addresses to canonical names.
+  - Adds, removes, or sets cname records for ip and hostname pairs.
+  - Modifies %windir%\\system32\\drivers\\etc\\hosts.
 options:
   state:
     description:
       - Whether the entry should be present or absent.
-      - If only C(canonical_name) is provided when C(state=absent), then
+      - If only I(canonical_name) is provided when I(state=absent), then
         all hosts entries with the canonical name of I(canonical_name)
         will be removed.
-      - If only C(ip_address) is provided when C(state=absent), then all
+      - If only I(ip_address) is provided when I(state=absent), then all
         hosts entries with the ip address of I(ip_address) will be removed.
-      - If C(ip_address) and C(canonical_name) are both omitted when
-        C(state=absent), then all hosts entries will be removed.
+      - If I(ip_address) and I(canonical_name) are both omitted when
+        I(state=absent), then all hosts entries will be removed.
     choices:
       - absent
       - present
     default: present
+    type: str
   canonical_name:
     description:
       - A canonical name for the host entry.
-      - required for C(state=present).
+      - required for I(state=present).
+    type: str
   ip_address:
     description:
       - The ip address for the host entry.
       - Can be either IPv4 (A record) or IPv6 (AAAA record).
-      - Required for C(state=present).
+      - Required for I(state=present).
+    type: str
   aliases:
     description:
       - A list of additional names (cname records) for the host entry.
-      - Only applicable when C(state=present).
+      - Only applicable when I(state=present).
+    type: list
   action:
     choices:
       - add
       - remove
       - set
     description:
-      - Controls the behavior of C(aliases).
-      - Only applicable when C(state=present).
+      - Controls the behavior of I(aliases).
+      - Only applicable when I(state=present).
       - If C(add), each alias in I(aliases) will be added to the host entry.
       - If C(set), each alias in I(aliases) will be added to the host entry,
         and other aliases will be removed from the entry.
     default: set
+    type: str
 author:
   - Micah Hunsberger (@mhunsber)
 notes:
+  - supports C(check_mode)
```

Comment:
No really necessary but if you want to keep it I would capitalise it and add a full stop.

Reply:
I'll just remove it, thanks.